### PR TITLE
Bugzilla 134106

### PR DESCRIPTION
**Pull request by traghavendra**
The setuptools package is causing error with pcs.
So, skipping it.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,8 +13,15 @@ LABEL name="rhosp16.1/openstack-cinder-volume-hpe" \
 # switch to root and install a custom RPM, etc.
 USER "root"
 
+
+RUN export http_proxy=http://web-proxy.atl.hpecorp.net:8080/ && export https_proxy=http://web-proxy.atl.hpecorp.net:8080/
```
Comment (sneharai4) [inline Dockerfile]: Shall we remove the proxy from Dockerfile?

File: README.md [hunk 2]
```diff
@@ -1,87 +1,142 @@
...
+
+IMPORTANT:
+
+The containers-prepare-parameter-hpe.yaml file replaces the standard containers-prepare-parameter.yaml file in your overcloud deployment. Do not include the standard containers-prepare-parameter.yaml file in your overcloud deployment. Retain the standard containers-prepare-parameter.yaml file for your undercloud installation and updates.
+
+
+
+#### 1.2 Environment File for cinder backend
+
+The environment file is a OSP director environment file. The environment file contains the settings for each backend you want to define.
```
Comment (sneharai4) [inline README.md]: ... is an* OSP ....

File: README.md [hunk 3]
```diff
@@ -1,87 +1,142 @@
...
-Create new env file “custom_container_[iscsi|fc].yaml” under /home/stack/custom_container/ with only the custom container parameter and other backend details. Sample files are available in [custom_container](https://github.com/hpe-storage/hpe-3par-cinder-rhosp16.1/blob/master/custom_container) folder for reference
 ```
 parameter_defaults:
-    DockerCinderVolumeImage: cld13b4.ctlplane.set.rdlabs.hpecorp.net:8787/rhosp-rhel8/openstack-cinder-volume-hpe:latest
+  CinderEnableIscsiBackend: false
+  ControllerExtraConfig:
 ```
 
-### 3.	Deploy the overcloud
+Sample files for both iSCSI and FC backends are available in [templates](https://github.com/traghavendra/hpe-3par-cinder-rhosp16.1/tree/main/templates) folder for reference.
```
Comment (sneharai4) [inline README.md]: Need to put master link here instead of forked repo user.

File: Dockerfile (hunk 1 above)
Comment (traghavendra) [reply Dockerfile]: done

File: README.md (hunk 2 above)
Comment (traghavendra) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (traghavendra) [reply README.md]: done

**Comment by traghavendra** [comment]
Updated as per review comments.

**Review by sneharai4 — APPROVED** [review]

**Pull request merged**